examples/chef/common/stubs.cpp

```diff
@@ -0,0 +1,16 @@
+#include <app-common/zap-generated/attribute-id.h>
+#include <app-common/zap-generated/attribute-type.h>
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/callback.h>
+#include <app-common/zap-generated/cluster-id.h>
+#include <app-common/zap-generated/command-id.h>
+
+bool emberAfPluginDoorLockOnDoorLockCommand(chip::EndpointId endpointId, chip::Optional<chip::ByteSpan> pinCode)
```

Comment:
why is door lock special here? why is the file called stubs.cpp not door_lock_stubs.cpp?

Some comment explaining why this file exist and what it is supposed to contain helps here. if this is named 'door_lock_stubs.cpp' that would not be needed, but if stubs it does make me wonder why lock is here but not others.